maven-jxr-plugin/src/main/java/org/apache/maven/plugin/jxr/AbstractJxrReport.java

```diff
@@ -457,21 +414,14 @@ private void copyResources( String dir, String sourceFolder, String... files )
     }
 
     @Override
-    protected Renderer getSiteRenderer()
+    protected MavenProject getProject()
```

Comment:
The same method in `AbstractMavenReport`

Reply:
True, but again in UTs the visibility in package is different.